pages/common/mtr.md

```diff
@@ -5,24 +5,28 @@
 
 - Traceroute to a host and continuously ping all intermediary hops:
 
-`mtr {{host}}`
+`mtr {{example.com}}`
```

Comment:
As per the title (which is long and maybe we could rephrase it to something more concise), we might have missed to update the homepage.

Reply:
Oops, I guess I forgot to push the change. Feel free to suggest something "better" for the title.